apps/website/src/app/components/HeroCarousel.tsx

```diff
@@ -37,18 +37,24 @@ const examples = [
 ];
 
 export async function HeroCarousel() {
-  const parsedNames = examples.map((n) => parseName(n));
-  const data = await nameguard.bulkInspectNames(
-    parsedNames.map((n) => n.outputName.name)
-  );
+  const ensNames = examples.map((n) => buildENSName(n));
+  const data = await nameguard.bulkInspectNames(examples);
 
   return (
     <div className="w-[200%] group flex flex-nowrap justify-center items-center space-x-1 animate-carousel group-hover:pause-on-hover">
       {data?.results?.map((report, index) => (
-        <HeroReportBadge key={`carousel-item-${index}`} data={report} />
+        <HeroReportBadge
+          key={`carousel-item-${index}`}
+          ensName={ensNames[index]}
+          data={report}
+        />
       ))}
       {data?.results?.map((report, index) => (
```

Comment:
I think this was added in the past in order to avoid duplicated names in the visible ones list... Probably this names list grew with time and now this duplication of comps is not anymore needed

Reply:
@FrancoAguzzi If this isn't needed anymore then let's please remove it. If it still is needed please add a comment explaining why it is needed.

Reply:
@FrancoAguzzi An open question for you here.